### ISSUE TITLE
Propfind-Response partially URL encoded

### ISSUE THREAD

**Issue by roroettg** [civetweb/civetweb#729]
Hi,

when I make propfind-Requests to the Server every href property gets encoded except for the requested directory itself. Is this a bug or just a misconfiguration on my side? Can I deactivate response encoding?

``` xml
<?xml version="1.0" encoding="utf-8"?>  
<d:multistatus xmlns:d='DAV:'>
    <d:response>
        <d:href>/webdav/DownloadFiles/</d:href>
        <d:propstat>
            <d:prop>
                <d:resourcetype>
                    <d:collection/>
                </d:resourcetype>
                <d:getcontentlength>1024</d:getcontentlength>
                <d:getlastmodified>Mon, 28 Jan 2019 12:17:39 GMT</d:getlastmodified>
            </d:prop>
            <d:status>HTTP/1.1 200 OK</d:status>
        </d:propstat>
    </d:response>
    <d:response>
        <d:href>%2fwebdav%2fDownloadFiles%2fxcvxcv</d:href>
        <d:propstat>
            <d:prop>
                <d:resourcetype>
                    <d:collection/>
                </d:resourcetype>
                <d:getcontentlength>1024</d:getcontentlength>
                <d:getlastmodified>Wed, 23 Jan 2019 14:13:29 GMT</d:getlastmodified>
            </d:prop>
            <d:status>HTTP/1.1 200 OK</d:status>
        </d:propstat>
    </d:response>
    <d:response>
        <d:href>%2fwebdav%2fDownloadFiles%2fsdsdf.png</d:href>
        <d:propstat>
            <d:prop>
                <d:resourcetype></d:resourcetype>
                <d:getcontentlength>152644</d:getcontentlength>
                <d:getlastmodified>Thu, 24 Jan 2019 16:49:33 GMT</d:getlastmodified>
            </d:prop>
            <d:status>HTTP/1.1 200 OK</d:status>
        </d:propstat>
    </d:response>
</d:multistatus>
```

**Comment by xtne6f** [comment]
What is your CivetWeb version?

**Comment by roroettg** [comment]
CivetWeb v1.11, built on Dec 12 2018

**Comment by xtne6f** [comment]
I see. It is a bug, and I have already fixed it by commit: 5e8d424458e682aac7ae0cdf1cdfac8d6f30ad6f .
Although the commit has been merged to the [master](https://github.com/civetweb/civetweb/tree/master) branch, CivetWeb v1.12 is not yet released...

**Comment by roroettg** [comment]
Ok, then this issue will be fixed in the next release.
